src/components/donation-form/AutofillDonorEmail.tsx

```diff
@@ -41,9 +41,6 @@ export default function AutofillDonorEmail(props: Donor) {
           label="Email Address"
           id="outlined-required"
           value={''}
-          onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
I don't really know why this is changed but let me know if it was intentional because I think it should be changed back